docs/en/integrations/language-clients/java/jdbc-v2.md

```diff
@@ -0,0 +1,141 @@
+import CodeBlock from '@theme/CodeBlock';
+
+
+# JDBC Driver
+
+`clickhouse-jdbc` implements the standard JDBC interface using [client-v2](/docs/en/integrations/language-clients/java/client-v2.md).
+We recommend using [client-v2](/docs/en/integrations/language-clients/java/client-v2.md) directly if performance/direct access is critical.
+
+## Environment requirements
+
+- [OpenJDK](https://openjdk.java.net) version >= 8
+
+
+### Setup
+
+<Tabs groupId="jdbc-base-dependencies">
+<TabItem value="maven" label="Maven" >
+
+```xml 
+<!-- https://mvnrepository.com/artifact/com.clickhouse/clickhouse-jdbc -->
+<dependency>
+    <groupId>com.clickhouse</groupId>
+    <artifactId>clickhouse-jdbc</artifactId>
+    <version>0.8.0</version>
+    <classifier>shaded-all</classifier>    
+</dependency>
+```
+
+</TabItem>
+<TabItem value="gradle-kt" label="Gradle (Kotlin)">
+
+```kotlin
+// https://mvnrepository.com/artifact/com.clickhouse/clickhouse-jdbc
+implementation("com.clickhouse:clickhouse-jdbc:0.8.0:shaded-all")
+```
+</TabItem>
+<TabItem value="gradle" label="Gradle">
+
+```groovy
+// https://mvnrepository.com/artifact/com.clickhouse/clickhouse-jdbc
+implementation 'com.clickhouse:clickhouse-jdbc:0.8.0:shaded-all'
+```
+
+</TabItem>
+</Tabs>
+
+## Configuration
+
+**Driver Class**: `com.clickhouse.jdbc.ClickHouseDriver`
+
+**URL Syntax**: `jdbc:(ch|clickhouse)[:<protocol>]://endpoint1[,endpoint2,...][/<database>][?param1=value1&param2=value2][#tag1,tag2,...]`, for example:
+
+- `jdbc:clickhouse:http://localhost:8123`
+- `jdbc:clickhouse:https://localhost:8443?ssl=true`
+
+**Connection Properties**:
+
+Beyond standard JDBC properties, the driver supports the ClickHouse-specific properties offered by the underlying [client](/docs/en/integrations/language-clients/java/client-v2.md).
+Where possible methods will return an SQLFeatureNotSupportedException if the feature is not supported. Other custom properties include:
+
+| Property                         | Default | Description                                                    |
+|----------------------------------|---------|----------------------------------------------------------------|
+| `disable_frameworks_detection`   | `true`  | Disable frameworks detection for User-Agent                    |
+| `jdbc_ignore_unsupported_values` | `false` | Suppresses SQLFeatureNotSupportedException                     |
+| `clickhouse.jdbc.v1`             | `false` | Use JDBC-V1 instead of JDBC-V2                                 |
+| `default_query_settings`         | `null`  | Allows passing of default query settings with query operations |
+
+## Supported data types
+
+JDBC Driver supports same data formats as the client library does. 
+
+:::note
+- AggregatedFunction - :warning: does not support `SELECT * FROM table ...`
+- Decimal - `SET output_format_decimal_trailing_zeros=1` in 21.9+ for consistency
+- Enum - can be treated as both string and integer
+- UInt64 - mapped to `long` (in client-v1) 
```

Comment:
What does `in client-v1` mean? It might be confusing to introduce the fourth concept on the page (jdbc v1/v2, client v1/v2)